src/app/shared/profile-data-view/profile-data-view.component.ts

```diff
@@ -695,10 +698,22 @@ export class ProfileDataViewComponent implements OnInit, OnChanges {
           const namespace = defaultProfileProperties[0].find(property => property.key === 'namespace');
           const name = defaultProfileProperties[0].find(property => property.key === 'name');
           const version = defaultProfileProperties[0].find(property => property.key === 'version');
-          if (namespace !== undefined && name !== undefined && version !== undefined) {
+          if (!this.useDefaultProfileSimplifiedEntry &&
+              namespace !== undefined && name !== undefined && version !== undefined) {
             responses = this.usedProfiles.filter(usedProfile =>
               usedProfile.namespace === namespace.value && usedProfile.name === name.value && usedProfile.version === version.value
             ).concat(responses);
+
+          } else {
+            for (const property_item of defaultProfileProperties[0]) {
+              if (property_item !== undefined ) {
+                responses = responses.concat(
+                  this.usedProfiles.filter(usedProfile =>
+                      ((decodeURIComponent(usedProfile.name) +'/'+ usedProfile.version) === property_item.value)
+                  )
+                );
```

Comment:
I've noticed that if a default profile using the `ui.default.profile.simplified_entry` flag is set at the top level (DataModel level), then a different default profile is set at the DataClass level, the DataModel level applies rather than the DataClass profile.

I think this should be 

```typescript
responses = this.usedProfiles.filter(usedProfile =>
  ((decodeURIComponent(usedProfile.name) + '/' + usedProfile.version) === property_item.value)
).concat(responses);
```

with the `concat` the other way round?